core/src/main/java/org/testcontainers/lifecycle/Startable.java

```diff
@@ -6,6 +6,8 @@
 
     void stop();
 
+    void restart();
```

Comment:
I would not introduce this method here. Also, it breaks the backward compatibility

Reply:
You mean backward compatibility with possible user implementations of `Startable`?